xml/cap_overview.xml

```diff
@@ -89,7 +89,8 @@
 
   <para>
    &productname; is a software platform for cloud-native application deployment
-   based on &scf; and &kube;.
+   based on &scf; and &kube;. Starting from version 2.0, it is based on
+   &operator; and &kubecf;.
```

Comment:
I'd suggest rearranging the sentence so that CAP2 is first as opposed to using the description of the CAP1 composition first and having the emphasis there.

Example suggestion would be: 

`&productname; is a software platform for cloud-native application deployment based on &operator; and &kubecf;. &productname; releases prior to version 2.0 were based on &scf; and &kube;.`

This applies to other work items as well, but do you think we should:
- Refer back to CAP1 in the CAP2 docs? 
- Limit the CAP1 references/differences to a small subsection?
- Discuss with other stakeholders?

Reply:
I guess it doesn't make much sense to refer to the previous versions in general text unless they're dedicated to that (release notes, troubleshooting, migration).